Parsers/ASimAuditEvent/Parsers/vimAuditEventSentinelOne.yaml

```diff
@@ -0,0 +1,584 @@
+                EventSubType = EventSubType_otheractivity,
+                Object = coalesce(Object, fileDisplayName, applicationPath, externalId, roleName, ruleName, incidentName, deviceId, recoveryEmail, featureName, policyName, tagName),
+                NewValue = coalesce(newIncidentStatusTitle, newTicketId, newAnalystVerdictTitle, newConfidenceLevel, newStatus, current, Status, newTagName, newTagDescription, featureValue),
+                OldValue = coalesce(oldIncidentStatusTitle, oldTicketId, oldAnalystVerdictTitle, oldConfidenceLevel, oldStatus, previous, oldTagName, oldTagDescription),
+                TargetIpAddr = coalesce(externalIp, ip, gatewayExternalIp),
+                ThreatCategory = threatClassification,
+                RuleName = ruleName,
+                TargetDvcId = deviceId
+            | invoke _ASIM_ResolveDstFQDN('affectedDevices')
+            | project-rename
+                TargetHostname = DstHostname,
+                TargetDomain = DstDomain,
+                TargetDomainType = DstDomainType,
+                TargetFQDN = DstFQDN,
+                TargetUrl = consoleUrl;
+        let parsedotheractivitydata_52565258 = rawotheractivitydata
+            | where activityType_d in (5256, 5258)
+            | extend EventType = case(
+                            isnotempty(rulesAdded) or isnotempty(tagsAdded),
+                            "Create",
+                            isnotempty(rulesRemoved) or isnotempty(tagsRemoved),
+                            "Delete",
+                            "Set"
+                        );
+        let parsedotheractivitydata_objectvalue = rawotheractivitydata
+            | where activityType_d in (3008, 3009, 3010, 3011, 3012, 3013, 3014, 3015, 3016, 3017, 3018, 3019, 3650, 3651, 3652, 3653, 3654)
+            | extend Object = strcat(Object, ' ', value);
+        union
+            groupsiteactivitydata,
+            machineactivitydata,
+            accountactivitydata,
+            useractivitydata,
+            rawotheractivitydata,
+            parsedotheractivitydata_52565258,
+            parsedotheractivitydata_objectvalue
+        | where (eventresult == "*" or EventResult =~ eventresult)
+            and (array_length(eventtype_in) == 0 or EventType has_any (eventtype_in))
+            and (array_length(operation_has_any) == 0 or Operation has_any (operation_has_any))
+            and (array_length(newvalue_has_any) == 0 or NewValue has_any (newvalue_has_any))
+            and (array_length(object_has_any) == 0 or Object has_any (object_has_any))
+            and (array_length(srcipaddr_has_any_prefix) == 0 or has_any_ipv4_prefix(ipAddress, srcipaddr_has_any_prefix))
+        | lookup EventSeverityLookup on EventResult
+        | lookup EventSeverityLookup_activity on activityType_d
+        | extend 
+            EventSeverity = coalesce(EventSeverity_activity, EventSeverity_lookup),
```

Comment:
Few logs having threat activity like Malware are also categorized as Informational. Can we re-check?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/7a7d49c4-2b8a-4721-b044-2ba873d30ae6)

Reply:
Updated